frontend/src/SocialMediaAccountsListTile.tsx

```diff
@@ -8,6 +8,11 @@ import { DeploymentManager } from "./DeploymentManager";
 import { Credential, AccountDetailsTile } from "./AccountDetailsTile";
 import { ToastManager } from "./ToastManager";
 import { EntityUtil } from "./EntityUtil";
+import { SnippetManager } from "./SnippetManager";
+
+export interface SocialMediaAccountsListTileProps {
+    eventBus: EventBus
+}
 
 export interface SocialMediaAccount {
```

Comment:
If `SocialMediaAccount` had an `idName` property (e.g. "twitter", "facebook", etc.), `getSocialmediaIcon()` could be simplified to just pass the `idName` to  `ImageProvider.getImage()`.